models/comments.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection")
+
+function fetchAllComments(article_id) {
+    return db.query(`SELECT * FROM comments WHERE comments.article_id=$1 ORDER BY comments.created_at DESC;`, [article_id])
+    .then(({rows: comments}) => {
+        if(comments.length === 0) {
+            return Promise.reject({status: 404, msg: "comment not found"})
```

Comment:
Sometimes, there will be no comments for an article and we wouldn't expect an error, rather we'd just expect to get an empty array and send that with a 200
As long as the article exists, we can send back whatever this query finds, empty or not
So we need to first check if the article exists before progressing with finding its comments